### PR TITLE
docs: add region tag for sigterm handler

### PR DESCRIPTION
**Pull request by averikitsch**
## Description

Fixes #<ISSUE-NUMBER>

Note: It's a good idea to open an issue first for discussion.

## Checklist
- [ ] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#readme-file)
- [ ] **Tests** pass:   `nox -s py-3.6` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] Please **merge** this PR for me once it is approved.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/.github/CODEOWNERS) with the codeowners for this sample

### CODE REVIEW

File: run/idp-sql/main.py [hunk 1]
```diff
@@ -97,11 +97,17 @@ def shutdown_handler(signal: int, frame: FrameType) -> None:
...
+# [END cloudrun_sigterm_handler]
+
 if __name__ == "__main__":
     # handles Ctrl-C locally
     signal.signal(signal.SIGINT, shutdown_handler)
 
     app.run(host="127.0.0.1", port=8080, debug=True)
 else:
     # handles Cloud Run container termination
+    # [START cloudrun_sigterm_handler]
```
Comment (grant) [inline run/idp-sql/main.py:109]: I'd expect the code comment for a statement be included with the statement.

Do we want to include the comments for the statements here and above to include the comment?

**Review by grant — APPROVED** [review]
1 comment.
LGTM.

File: run/idp-sql/main.py (hunk 1 above)
Comment (averikitsch) [reply run/idp-sql/main.py:109]: I specifically excluded them since they aren't too helpful out of context.

**Comment by averikitsch** [comment]
ping @leahecole

**Review by leahecole — APPROVED** [review]

**Pull request merged**